prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-overview/enrollee-overview.component.ts

```diff
@@ -36,6 +36,7 @@ export class EnrolleeOverviewComponent extends AdjudicationContainerComponent im
   public showAdjudication: boolean;
   public documents: EnrolleeAdjudicationDocument[];
   public absence: EnrolleeAbsence;
+  public readonly paperEnrolleeGpidFilter = 'NOBCSC';
```

Comment:
In the front end we have a constant.ts. Probably time to move this into that file and in the places we use it for logic we should use it from there.